techmatters/terraso-mobile-client
# Feat: Home info screen: new text, image, and external link

## Description
- Style the component for external links (LinkNewWindowIcon.tsx)
- Refactor to move icon components to their own files (this is why there are so many files)
- Update home info sheet to match Figma design (LandPKSInfo.tsx)
- (Also updated the empty sites list screen to remove the link... also updated content and noted an issue with BulletList layout, which was tangential to the Issues this primarily addresses, whoops.) 

### Checklist

### Related Issues
Implements #801, #954

### Verification steps
Confirm that the info sheet on the home page looks like the Figma design
Confirm that the styling of the external link(s) in the LandPKS Data Privacy screen looks correct
(Also, confirm that the empty sites list looks reasonable / like the Figma as of 2024-04-24)
